### DISCUSSION TITLE
No daytime image on remote website

### DISCUSSION THREAD

**Discussion by jk1974-tvh** [aaronwmorris/indi-allsky#1131] [Q&A]
On my remote website (which is the one that was set up with a previous installation of Thomas Jacquin's allsky software) I only get the Aurora Forecast displayed during the day, plus a message that it's not yet dark in my place. How can I get the page to display the current image of my camera?

**Comment by aaronwmorris** [comment]
Are you using indi-allsky on the remote site?

**Reply by jk1974-tvh** [reply]
Hi Aaron,
thank you for the quick reply. I'm running indi-allsky on a pi4, the remote website is self-hosted on a Synology Diskstation. I upload videos and startails to the folders created by the previous allsky Installation. I get "live" nighttime images shown on the remote website but no daytime image. 
Would the install process for the remote site be the same as the one on the pi?

**Reply by jk1974-tvh** [reply]
I also get only corrupted video files and no thumbnails are created.

**Comment by aaronwmorris** [comment]
If you have daytime timelapse disabled, then images will not be transferred to a remote site.  That is the only reason I can think this behavior would occur.
